mkinitcpio-install.sh

```diff
@@ -52,6 +52,33 @@ run_command() {
     esac
 }
 
+concatenate_unit() {
+    # concatenate units and their potential drop-in files
+    # we do this manually, as in a chroot `systemctl cat` will not work
+    local unit_name="$1"
+    local unit_content=""
+    # add the top-most service
```

Comment:
local service_path=

Reply:
What are these changes for?  Just a rename of the variables?

Reply:
declare these function-local to prevent namespace leak, 
since we work inside larger context of `mkinitcpio` build invocation

Reply:
Ah